.evergreen/compile-artifact.sh

```diff
@@ -96,10 +96,10 @@ if uname -a | grep -q 'Linux.*x86_64'; then
   test $(objdump -d dist/mongosh | grep '\bvmovd\b' | wc -l) -lt 1250
 fi
 
-npm run write-node-js-dep
-npm run create-purls-file
-cp .sbom/purls.txt dist/.purls.txt
-
-cat dist/.purls.txt
-
 npm run create-dependency-sbom-lists
+
+ls -lhA .sbom
+for dir in .sbom/*/; do
+  cp ${dir}purls.txt dist/${dir}purls.txt
+  cat dist/${dir}purls.txt
+done
```

Comment:
How does this work when publishing packages through GHA?

The ticket was created before that was a thing, but now I think we'll need to account for that – maybe it could be encapsulated so that it only really affects the compile step of the node-runtime-worker-thread package? That would mean missing out on Silk/Kondukto integration for it, but I think that's okay given that it's not a standalone supported product

Reply:
Good point; will look into potentially moving this to the build step instead